syntax = "proto3";

package ltproto.peer2peer;
option java_package = "cn.lanthing.proto";

message VideoFrame
{
    bool is_keyframe = 1;
    uint64 picture_id = 2;
    uint32 width = 3;
    uint32 height = 4;
    int64 capture_timestamp_us = 5;
    int64 start_encode_timestamp_us = 6;
    int64 end_encode_timestamp_us = 7;
    optional uint32 temporal_id = 8;
    bytes frame = 9;
}
